src/app/store.ts

```diff
@@ -1,10 +1,18 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
```

Comment:
you shouldn't disable this eslint rule, pls refactor your code so that you won't have to disable it